packages/editor/src/components/widgets/table/table/Table.css

```diff
@@ -19,3 +18,15 @@
 .table tr :where(td) {
   border-top: var(--seperator-border);
 }
+
+.table tr:last-child :where(td:first-child) {
```

Comment:
where is only needed if you use more than one selector in it
reduce complication -> `.table tr:last-child td:first-child {`